Docs/Request_Overrides.md

```diff
@@ -179,5 +179,78 @@ Example:
     "DEVICE_TYPE": "ipstb",
     "DEVICE_MODEL": "VALUE",
      ...
-}
+  }
   ```
+
+
+## screenshot:
+
+### Request override
+- The `screenshot` request override is used to take a screenshot of the current screen of the platform. It is added in the config module `requestModules/fcs.js` file.
+
```

Comment:
This function has to be added in config module `requestModules/fcs.js` file.

Reply:
Updated